contracts/senders/aave-v2-migrator/variables.sol

```diff
@@ -71,13 +72,39 @@ contract Variables {
      */
     StateSenderInterface constant internal stateSender = StateSenderInterface(0x28e4F3a7f651294B9564800b2D01f35189A5bFbE);
 
-
-    // InstaIndex Address.
+    /**
+     * @dev InstaIndex Address.
+     */
     IndexInterface public constant instaIndex = IndexInterface(0x2971AdFa57b20E5a416aE5a708A8655A9c74f723);
 
     /**
-     * Polygon deposit bridge
+     * @dev Polygon deposit bridge
      */
     RootChainManagerInterface public constant rootChainManager = RootChainManagerInterface(0xA0c68C638235ee32657e8f720a23ceC1bFc77C77);
+    
+    
+    // Storage variables //
+    
+    /**
+    * @dev This will be used to have debt/collateral ratio always 20% less than liquidation
+    */
+    // TODO @KaymasJain: Is this number correct for it?
+    uint public safeRatioGap = 800000000000000000; // 20%?
```

Comment:
Yes! It's correct. We're taking 100% of the position and multiplying it by 80%.

Reply:
```suggestion
    uint public safeRatioGap = 800000000000000000; // 80%
```